### ISSUE TITLE
add notes to alias

### ISSUE THREAD

**Issue by mackxu** [ShivanshRakesh/url-aliaser#10]
The alias may be forgotten after a while, I hope to add optional remarks for each record, thank you

**Comment by ShivanshRakesh** [comment]
Yeah, that sounds like a good idea. Are you willing to work on this?
